tests/on-device/config-h-m.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, config, ... }:
+
+{
+  system.stateVersion = "21.05";
+
+  home-manager.config =
+    { pkgs, lib, ... }:
+    {
+      home.stateVersion = "21.05";
+      nixpkgs.overlays = lib.mkIf (lib.hasAttrByPath [ "nixpkgs" "overlays" ] config) config.nixpkgs.overlays;
```

Comment:
Why not enable `home-manager.useGlobalPkgs` instead of this monstrosity? I am still not sure if that complexity is really necessary for setting just the overlay :D

Reply:
Shouldn't be needed for 21.11 anymore